src/veh_interact.cpp

```diff
@@ -2092,6 +2083,16 @@ void veh_interact::display_details( const vpart_info *part )
         }
     }
 
+    if( part->has_flag( VPFLAG_WHEEL ) ) {
+        cata::optional<islot_wheel> whl = item::find_type( part->item )->wheel;
+        fold_and_print( w_details, line+3, col_1, column_width, c_white,
+                            "%s: <color_light_gray>%d\"</color>", small_mode ? _( "Dia" ) : _( "Wheel Diameter" ),
+                            whl->diameter );
+        fold_and_print( w_details, line+3, col_2, column_width, c_white,
+                            "%s: <color_light_gray>%d</color>", small_mode ? _( "Wdt" ) : _( "Wheel Width" ),
```

Comment:
aaand very last comment before merge: width dimension needs units, ie ```<color_light_gray>%d</color>``` (with ```"``` symbol, like in diameter formatting above)